src/components/header/Header.tsx

```diff
@@ -14,16 +27,27 @@ export const Header = (): JSX.Element => {
             <Link to='/'>Home</Link>
           </li>
           <li className='header__link'>
-            {currentUser ? (
-              <span>Log out</span>
+            {customer ? (
+              <div className='header__profile-logout'>
+                <Link to='/profile'>
```

Comment:
Thank you for creating './profile' path link. We will add it to the router path in Sprint 3